src/components/BotBuilder/BotBuilderWrap.js

```diff
@@ -35,39 +35,34 @@ const templates = [
   },
 ];
 
-class BotBuilderWrap extends React.Component {
-  componentDidMount() {
-    document.title = 'SUSI.AI - Botbuilder';
-  }
-  render() {
-    if (!cookies.get('loggedIn')) {
-      return (
-        <div>
-          <StaticAppBar {...this.props} />
-          <div>
-            <p style={styles.loggedInError}>
-              Please login to create a chatbot.
-            </p>
-          </div>
-        </div>
-      );
-    }
+const BotBuilderWrap = props => {
+  document.title = 'SUSI.AI - Botbuilder';
+  if (!cookies.get('loggedIn')) {
```

Comment:
How about using one return statement and a ternary if operator. 

Eg: 
```
return isLoggedin ? (<div>1</div>) : (<div>2</div>)

With the linter on, this would look much better too.

Reply:
@AakashMallik it's a multiline return statement